net/nfs-kernel-server/Makefile

```diff
@@ -37,96 +37,120 @@ define Package/nfs-kernel-server/Default
 	URL:=http://nfs.sourceforge.net/
 endef
 
+define Package/nfs-utils/Default
+	SECTION:=utils
+	CATEGORY:=Utilities
+	SUBMENU:=Filesystem
+	URL:=http://nfs.sourceforge.net/
+endef
+
 define Package/nfs-kernel-server
 	$(call Package/nfs-kernel-server/Default)
-	TITLE:=Kernel NFS server support
-	DEPENDS+= +kmod-fs-nfsd +kmod-fs-nfs +NFS_KERNEL_SERVER_V4:kmod-fs-nfs-v4 +rpcbind +NFS_KERNEL_SERVER_V4:nfs-utils-libs +NFS_KERNEL_SERVER_V4:libkeyutils +NFS_KERNEL_SERVER_V4:libdevmapper
+	TITLE:=Kernel NFS server (NFSv3)
+	DEPENDS+= +kmod-fs-nfsd +kmod-fs-nfs +rpcbind
+	VARIANT:=v3
+	PROVIDES:=nfs-kernel-server
```

Comment:
```suggestion
```
As the package is called `nfs-kernel-server` it doesn't need to provide `nfs-kernel-server`